packages/common/src/types/semanticLayer.ts

```diff
@@ -0,0 +1,18 @@
+import { type FieldType } from './field';
+
+export type SemanticLayerView = {
+    name: string;
+    label: string;
+    description?: string;
+    visible?: boolean;
```

Comment:
what's visible for? did that come from an api?

Reply:
yes, visible is a property from the api. We already know some fields are going to be `not visible` if some other dimensions are selected, and we are going to `disable` it in the new `view` page.